app/src/main/res/layout-sw600dp-land/content_item.xml

```diff
@@ -30,7 +46,7 @@
     app:explorationSplitViewPaddingTop="@{@dimen/content_item_exploration_split_view_padding_top}"
     app:explorationViewMarginApplicable="@{viewModel.hasConversationView &amp;&amp; !viewModel.isSplitView}"
     app:explorationViewMarginBottom="@{@dimen/space_0dp}"
-    app:explorationViewMarginEnd="@{@dimen/content_item_exploration_margin_end}"
```

Comment:
This dimen was resulting #2564 error. Removed this and replaced this with `content_item_exploration_view_margin_end `

Reply:
Just to be clear, You had to remove it because both of them had the use for exploration view end margin but mistakenly two copies of this were created with different names and values which broke the UI right?

Reply:
Yes. and both those variables had different values.